scrapers/nytData.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const csv = require('csvtojson');
+const logger = require('../utils/logger');
+const { updateCache } = require('../utils/nyt_cache');
+
+const US_COUNTY_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv';
+const US_STATE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv';
+const US_NATION_WIDE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us.csv';
+const REDIS_KEYS = ['nyt_counties', 'nyt_states', 'nyt_USA'];
+
+/**
+ * Retrieves NYT data from github and stores it in redis
+ * @param {Object} keys Redis keys sourced from configurations
+ * @param {Object} redis Redis instance
+ */
+const nytData = async (keys, redis) => {
+	try {
+		const _resolveData = async (url, index) => {
+			const { data } = await axios.get(url);
```

Comment:
maybe add checking response code like we do in the other scraper code, i dont know, i like your way too, but we kinda should agree on one way

Reply:
Not sure what you mean exactly, can you link to the code in the scraper code that you're referring to?

Reply:
okay seems i was the one to ditch this in my last refactoring, so forget about it, i love how did that object destructing

Reply:
seems i was the one to ditch this in the last refactor haha, forget about it, maybe make a try catch around (like in getWorldometerPage and then call the logger if an err occurs)
love the object deconstruction!!